nautobot_golden_config/api/urls.py

```diff
@@ -36,4 +36,5 @@
         name="git_repository_branches",
     ),
 ]
+app_name = "nautobot_golden_config-api"
```

Comment:
What is this for?

Reply:
It prevents Django URL resolution conflicts, and is checked for during the app loading in Nautobot.